services/openuserjs/openuserjs-license.service.js

```diff
@@ -0,0 +1,38 @@
+import { renderLicenseBadge } from '../licenses.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSLicense extends BaseOpenUserJSService {
+  static category = 'license'
+  static route = { base: 'openuserjs', pattern: 'l/:username/:scriptname' }
+
+  static examples = [
+    {
+      title: 'OpenUserJS',
+      namedParams: {
+        username: 'NatoBoram',
+        scriptname: 'YouTube_Comment_Blacklist',
+      },
+      staticPreview: renderLicenseBadge({ licenses: ['GPL-3.0-or-later'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  transform(data) {
+    if (!('license' in data.UserScript)) {
+      // By the TOS, the script will be under an MIT License by default if not specified
+      // @see https://github.com/badges/shields/pull/8081#discussion_r898437399
+      return { licenses: ['MIT'] }
+    }
+    const licenses = data.UserScript.license.map(
+      license => license.value.split('; ')[0]
+    )
+    return { licenses: licenses.reverse() }
+  }
```

Comment:
Similar comment here as one I mentioned on another of your PRs, but this `transform` function has enough going on that I think unit tests would be helpful for us to have going forward. I can share some examples of how we do this if you aren't already familiar